config/default/manager/manager.yaml

```diff
@@ -158,8 +158,10 @@ webhooks:
         values:
           - "0"
           - "1"
-      matchLabels:
-        mutatepods.kubemacpool.io: allocate
+      - key: mutatepods.kubemacpool.io
```

Comment:
Do we need these at all? It feels it would be easier to grasp if the default without selectors completely and then we would require people building manifests to knowingly select one of the patches.

Reply:
If we don't add this then user can't manually opt-out, and I think it's important to be able to do that.

Reply:
The default will be only added in release/test. The logic would be, you get to choose what is the default, we won't make that for you.

Reply:
oh you mean only to remove it form the default/manager/manager.yaml. Sure.

Reply:
Do you think that  mutatevirtualmachines.kubemacpool.io should also be selector-free here? 
If so, I'll add patches for it as well

Reply:
I do. This should be symmetrical. We should make no difference between VM and Pod to the highest level possible.